log-ffi/src/lib.rs

```diff
@@ -16,7 +16,7 @@ use ddlog::logger::{logger_init, logger_set_log_level, LogCallback, LogLevel};
 ///
 /// Returns `None` on success, or a boxed `Error` if the update fails.
 #[no_mangle]
-pub extern "C" fn ddog_logger_set_log_level(log_level: LogLevel) -> Option<Box<Error>> {
+pub extern "C" fn ddog_log_logger_set_log_level(log_level: LogLevel) -> Option<Box<Error>> {
```

Comment:
Since this does not take a "logger" object, you could probably simplify the name to `ddog_log_set_log_level`. Same with `ddog_log_logger_init` to `ddog_log_init`.

Reply:
fair enough.